copper-server-api/src/main/java/com/pogeyan/cmis/api/utils/Helpers.java

```diff
@@ -62,6 +63,9 @@
 			"isIncludedInSupertypeQuery", "isControllablePolicy", "typeMutability", "isControllableAcl",
 			"propertyDefinition.cmis:name", "propertyDefinition.cmis:objectTypeId" };
 
+	public static String[] basicTypes = { "cmis:folder", "cmis:document", "cmis:item", "cmis:policy",
```

Comment:
Instead of hard coding , make use of that enum class BaseTypeId

Reply:
changed to enum class BaseTypeId and CustomTypeId